app/models/service_template_ansible_playbook.rb

```diff
@@ -193,10 +193,8 @@ def create_dialogs(config_info)
   end
 
   def delete_job_templates(job_templates)
```

Comment:
@jameswnl - Running this scenario:

1. Create new Ansible Playbook service template - no retirement playbook
2. Edit the catalog item - copy the provision settings over to the retirement.
3. Edit the catalog item - remove the playbook from the retirement tab.
4. Remove the entire catalog item.

1 - 3 worked as expected - 4 failed with this error: `Service Catalog Item "<catalog item nam>": Error during 'destroy': undefined method `delete_in_provider_queue' for nil:NilClass`